e2e/vm/cred_helper_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package vm
+
+import (
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"github.com/runfinch/common-tests/command"
+	"github.com/runfinch/common-tests/option"
+)
+
+var testCredHelper = func(o *option.Option, installed bool, registry string) {
+	ginkgo.Describe("Credential Helper", func() {
+		ginkgo.Describe("Credential Helper", func() {
+			ginkgo.It("should pull from container registry", func() {
+				resetVM(o, installed)
+				resetDisks(o, installed)
+				if registry == "" {
+					ginkgo.Skip("No Provided Container Registry Url")
+				}
+				writeFile(finchConfigFilePath, []byte("memory: 4GiB\ncredsHelper: ecr-login\ncpus: 6\nvmType: vz\nrosetta: false"))
+				cfgJSONPath := strings.ReplaceAll(finchConfigFilePath, "finch.yaml", "config.json")
```

Comment:
Shouldn't we be checking if Finch updates the config.json path itself rather than just setting it here?